src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
 // write your code here
+const successMessagePromise = new Promise((resolve) => {
+  const successMessage = document.createElement('div');
+
+  successMessage.setAttribute('class', 'message');
+
+  successMessage.textContent = 'Promise was resolved!';
+  resolve(successMessage);
+});
+
+successMessagePromise.then((successMessage) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    document.body.appendChild(successMessage);
+  });
+});
```

Comment:
```suggestion
const successMessagePromise = new Promise((resolve) => {
  document.querySelector('.logo').addEventListener('click', () => {
    resolve('Promise was resolved!');
  });
});

successMessagePromise.then((successMessage) => {
  printMessage(successMessage);
});
```

you can also create the common print function and move logic with printing the message there, but the main point here that all async logic should be inside promises